expenses_management_tool/streamlit/dashboard.py

```diff
@@ -0,0 +1,10 @@
+import streamlit as st
+
+# import pandas as pd
+
+# st.markdown("main dashboard")
+st.sidebar.markdown("main dashboard")
+
+st.title("Financial tracker")
```

Comment:
```suggestion
st.title("expense_management_tool")
```
is this the correct name for our project?

Reply:
Yes I agree that is our project's name. It's just that I prefer shorter and catchy name hahaha. The title can be change but I suggest we should just write it without `snake_case` format.

```suggestion
`st.title("Expense management tool")`
```

Just to confirm, which name are you prefer to use? Other name can be used too if you have another idea in mind.